libraries/aws_route_table_association.rb

```diff
@@ -0,0 +1,57 @@
+class AWSRouteTableAssociation < Inspec.resource(1)
+  name 'aws_route_table_association'
+  desc 'Test the configuration of a class AWS Route Table Association'
+  example "
+    describe aws_vpc_endpoint('rtb-1234', 'rtbassoc-1234') do
```

Comment:
You need to update that resource name :)

```suggestion
    describe aws_route_table_association('rtb-1234', 'rtbassoc-1234') do
```

Also, as it's kind of a special resource, can you actually add a smaller example, but that shows how to use wrt to the other resources? Something like you did for the real VPC endpoints where you want to test a route table of a VPC is configured with "public" subnets, or that it's the main route.
I think it would make the example a bit more clear in which context it's expected to be used.